src/utils/Observable.js

```diff
@@ -64,11 +76,11 @@ export default class Observable {
 
 	/**
 	 * @public
-	 * Destorys this observable, removes events and so on 
+	 * Destorys this observable, removes events and so on
```

Comment:
Great, that there is no trailing whitespace anymore. Would be nice to fix the "destorys" typo as well :)